src/main/java/teammates/common/util/Templates.java

```diff
@@ -51,6 +51,10 @@ public static String populateTemplate(String template, String... values) {
                 FileHelper.readResourceFile("userEmailTemplate-feedbackSessionUnpublished.html");
         public static final String USER_PENDING_COMMENTS_CLEARED =
                 FileHelper.readResourceFile("userEmailTemplate-pendingCommentsCleared.html");
+        public static final String USER_FEEDBACK_SESSION_FRAGMENT_RESEND_ALL_LINKS =
+                FileHelper.readResourceFile("userEmailTemplateFragment-feedbackSessionResendAllLinks.html");
+        public static final String USER_FEEDBACK_SESSION_RESEND_ALL_LINKS =
+                FileHelper.readResourceFile("userEmailTemplate-feedbackSessionResendAllLinks.html");
```

Comment:
The difference between this one and the one above is not immediately clear. Likely the first one should be `FRAGMENT_SINGLE_FEEDBACK_SESSION_LINKS` or alike to depict that it involves the links of one and only one feedback session.